src/test/java/learnositysdk/request/InitTest.java

```diff
@@ -451,6 +451,29 @@ public void testJsonSlashEscaping()
         );
     }
 
+	@Test
+	public void testJsonEnDashEscaping()
```

Comment:
Sorry I didn't catch this earlier, but it's probably better to name tests based on the general problem they're checking for. This way if it starts failing, we have a better idea of why we cared about this test in the first place: we want to unescape unicode characters in general, not just specifically en dashes.

Reply:
Hi @flakeparadigm I renamed the test to be more accurate.